runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsPlugin.java

```diff
@@ -311,6 +312,29 @@ public void onNpcSpawned(NpcSpawned npcSpawned)
 		}
 	}
 
+	@Subscribe
+	public void onNpcCompositionChanged(NpcCompositionChanged event)
```

Comment:
You should probably make common method for body of this event and use it in both npcspawned and here